### PR TITLE
Fix: #21. No specific YAML version.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- [ ] no specific YAML version

***
<a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/pull/41.html" title="Last updated on Jul 6, 2022, 8:56 PM UTC (7342d5b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/41/f9fd301...7342d5b.html" title="Last updated on Jul 6, 2022, 8:56 PM UTC (7342d5b)">Diff</a>

### CODE REVIEW

**Review by gkellogg — APPROVED** [review]

**Review by gkellogg — APPROVED** [review]

**Comment by gkellogg** [comment]
See comments in https://github.com/json-ld/yaml-ld/commit/2c26c46f3ab4e07f01988164d3d166119456f855#commitcomment-77653207. Best to comment on PRs rather than individual commits, IMO.

File: spec/index.html [hunk 1]
```diff
@@ -283,6 +283,14 @@ <h2>Introduction</h2>
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
+    <p>
+      This document is based on YAML 1.2.2,
+      but YAML-LD is not tied to a specific version of YAML.
```
Comment (gkellogg) [inline spec/index.html:288]: Should be a bibliographic reference.

File: spec/index.html [hunk 2]
```diff
@@ -283,6 +283,14 @@ <h2>Introduction</h2>
...
+      Implementers concerned about features related to a specific YAML version
+      can specify it in documents using the `%YAML` directive
+      (see <a href="#int" class="sectionRef"></a>).
+    </p>
+
```
Comment (gkellogg) [inline spec/index.html:293]: From https://github.com/json-ld/yaml-ld/commit/2c26c46f3ab4e07f01988164d3d166119456f855#r77665535, add something about transitive reflection of errors and warnings when parsing YAML.

File: spec/index.html (hunk 1 above)
Comment (ioggstream) [reply spec/index.html:288]: Isn't [[YAML]] enough? If not, can you show me where to add it?

File: spec/index.html [hunk 3]
```diff
@@ -301,7 +309,14 @@ <h2>Interoperability Considerations</h2>
       JSON documents in YAML, see [[YAML]]
       and the Interoperability consideration of application/yaml [[I-D.ietf-httpapi-yaml-mediatypes]]..
     </p>
-        
+    <p>
+      The YAML-LD format and the media type registration are not restricted to a specific
+      version of YAML,
+      but implementers that want to use YAML-LD with past or future versions of YAML
+      respect to 1.2.2 need to be aware that the considerations and analysis provided
```
Comment (TallTed) [inline spec/index.html:316]:
```suggestion
      but implementers that want to use YAML-LD with YAML versions
      other than 1.2.2 need to be aware that the considerations and analysis provided
```

File: spec/index.html (hunk 1 above)
Comment (gkellogg) [reply spec/index.html:288]: > Isn't [[YAML]] enough? If not, can you show me where to add it?

That should do it. SpecRef links it to http://yaml.org/spec/1.2/spec.html.

Comment (gkellogg) [inline spec/index.html:288]:
```suggestion
      but YAML-LD is not tied to a specific version of [[YAML]].
```

File: spec/index.html [hunk 4]
```diff
@@ -301,7 +309,14 @@ <h2>Interoperability Considerations</h2>
...
+      here, including interoperability and security considerations are based
```
Comment (TallTed) [inline spec/index.html:317]:
```suggestion
      here, including interoperability and security considerations, are based
```

**Review by OR13 — APPROVED** [review]

**Comment by gkellogg** [comment]
This PR was discussed in the [2022-07-06 meeting](https://json-ld.org/minutes/2022-07-06/), and it was agreed to merge.

**Pull request merged**